chef/cookbooks/provisioner/recipes/dhcp_update.rb

```diff
@@ -4,9 +4,12 @@
 admin_ip = Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "admin").address
 admin_net = node[:network][:networks]["admin"]
 lease_time = node[:provisioner][:dhcp]["lease-time"]
-pool_opts = {
-  "dhcp" => ['allow unknown-clients',
-             '      if option arch = 00:06 {
+
+pool_opts = {}
+if node[:provisioner][:dhcp]["for-unknown-nodes"]
+
+pool_opts["dhcp"] = ['allow unknown-clients',
```

Comment:
@vuntz: both fixed in https://github.com/toabctl/barclamp-provisioner/commit/4cc31e30cfab616a3a1938db92d523f0db7ce0b7